services/director-v2/tests/mocks/fake_task.json

```diff
@@ -61,6 +61,7 @@
   "created": "1961-07-06T11:24:30.877Z",
   "modified": "2008-03-24T07:02:09.279Z",
   "last_heartbeat": null,
+  "pricing_info": null,
```

Comment:
And now the the hidden issue here. What about existing data. Stored soemwhere in a DB maybe? Would this cause issues during the release if pricing_info is not automatically set as None.

@sanderegg you might have a better idea here?

Reply:
check the model of the DB. and check the values if it is not clear

Reply:
`pricing_info` is a required-nullable field in the Model. This is a mock and the field was missing. I don't get the issue...